crates/rattler_lock/src/parse/v3.rs

```diff
@@ -170,7 +170,7 @@ pub fn parse_v3_or_lower(
                             timestamp: value.timestamp,
                             track_features: value.track_features,
                             version: value.version,
-                            purls: value.purls,
+                            purls: Some(value.purls),
```

Comment:
I think this should be `None` if `value.purls` is empty. Basically the snapshots shouldn't really chance because the current state of affairs is that if the purls are not present they are unknown.

Reply:
changed